connect.go

```diff
@@ -172,9 +173,27 @@ func (r *Request[_]) Header() http.Header {
 	return r.header
 }
 
+// Method returns the HTTP method for this request. This is nearly always
+// POST, but side-effect-free RPCs could be made via a GET.
+//
+// On a newly created request, via NewRequest, this will return "POST" and
+// only changes to return to "GET" after it is actually sent to a server
+// using GET as the method.
+func (r *Request[_]) Method() string {
```

Comment:
The `Request` struct is also used for handlers, but I do not think `r.method` is being set by handlers in this PR. I think we either need to do that or document that this won't be set for handlers?

Reply:
Somehow I missed the actual place where this gets set, nevermind.